commitizen/commands/init.py

```diff
@@ -13,59 +13,150 @@
 from commitizen.defaults import config_files
 from commitizen.exceptions import InitFailedError, NoAnswersError
 from commitizen.git import get_latest_tag_name, get_tag_names, smart_open
+from commitizen.version_types import VERSION_TYPES
+
+
+class ProjectInfo:
+    """Discover information about the current folder."""
+
+    @property
+    def has_pyproject(self) -> bool:
+        return os.path.isfile("pyproject.toml")
+
+    @property
+    def has_setup(self) -> bool:
+        return os.path.isfile("setup.py")
+
+    @property
+    def has_pre_commit_config(self) -> bool:
+        return os.path.isfile(".pre-commit-config.yaml")
+
+    @property
+    def is_python_poetry(self) -> bool:
+        if not self.has_pyproject:
+            return False
+        with open("pyproject.toml") as f:
+            return "tool.poetry.version" in f.read()
+
+    @property
+    def is_python(self) -> bool:
+        return self.has_pyproject or self.has_setup
+
+    @property
+    def is_rust_cargo(self) -> bool:
+        return os.path.isfile("Cargo.toml")
+
+    @property
+    def is_npm_package(self) -> bool:
+        return os.path.isfile("package.json")
+
+    @property
+    def is_php_composer(self) -> bool:
+        return os.path.isfile("composer.json")
+
+    @property
+    def latest_tag(self) -> Optional[str]:
+        return get_latest_tag_name()
+
+    def tags(self) -> Optional[List]:
+        """Not a property, only use if necessary"""
+        if self.latest_tag is None:
+            return None
+        return get_tag_names()
+
+    @property
+    def is_pre_commit_installed(self) -> bool:
+        return not shutil.which("pre-commit")
 
 
 class Init:
     def __init__(self, config: BaseConfig, *args):
         self.config: BaseConfig = config
         self.cz = factory.commiter_factory(self.config)
+        self.project_info = ProjectInfo()
 
     def __call__(self):
         if self.config.path:
             out.line(f"Config file {self.config.path} already exists")
             return
 
-        # No config for commitizen exist
-        config_path = self._ask_config_path()
+        out.info("Welcome to commitizen!\n")
+        out.line(
+            "The prompts will ask you different questions " "to configure your project."
+        )
+        out.line("For further configuration visit:")
+        out.line("\n\thttps://commitizen-tools.github.io/commitizen/config/\n")
```

Comment:
out.line(
            "The prompts will ask you different questions to configure your project.\n"
            "For further configuration visit:\n"
            "\n"
            "https://commitizen-tools.github.io/commitizen/config/"
        )